sdk/eventhub/event-hubs/package.json

```diff
@@ -47,16 +47,16 @@
     "coverage": "nyc --reporter=lcov --exclude-after-remap=false mocha -t 120000 test-dist/index.js --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "echo skipped",
+    "integration-test:browser": "karma start --single-run",
     "integration-test:node": "mocha -t 1200000 test-dist/index.js --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint -c ../../.eslintrc.json src test samples --ext .ts --fix",
     "lint": "eslint -c ../../.eslintrc.json src test samples --ext .ts -f html -o event-hubs-lintReport.html || exit 0",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "test:browser": "npm run build:test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test:node && npm run integration-test:node",
```

Comment:
Why are we removing the call to the unit test related script here?

Reply:
The test scripts are not run on CI and are meant for dev use. See comments below from Brandon.

Reply:
Brandon's comment below actually __does__ include the `unit-test` scripts in the `test` script. It makes sense to run unit-tests as part of the developer's flow.

Reply:
That still doesnt explain why we are touching these scripts as part of the PR that is meant to enable browser tests in CI.

Yes, these are meant for dev use and are up to the devs on the Event Hubs project to decide what goes in here. As far as I recall, there has been no conversations on what the Event Hubs devs want these to be or what they want to remove from here.

Reply:
I would expect this PR to touch only the scripts relevant to building and testing the browser tests